std/eval/_std/sys/ssl/Certificate.hx

```diff
@@ -90,9 +90,7 @@ class Certificate {
 		return subject("CN");
 	}
 
-	private function get_altNames():Array<String> {
-		throw "Not implemented";
-	}
+	extern private function get_altNames():Array<String>;
```

Comment:
I don't know if this actually works because no Certificate I come across has a value for this, so all we get are empty arrays.